Task/01 setup

VasylBiliak opened the pull request.

Commit:
Added photo, styles and font

Commit:
Fixed a linter bug

Commit:
Fixed a linter bug

VasylBiliak commented:
- [x]  Mentor is added to pr
- [x]  AC copied to pr description
- [x]  link to deployed version added to pr
- [x]  About block (text and image) added to the main page
- [ ]  On the widths 390px and 1280px, the site layout corresponds to the design

MZabolotnev reviewed (changes requested):
Pls check deployment - its failed

VasylBiliak committed:
Update tsconfig.json

MZabolotnev commented on src/components/about/About.component.tsx:
```diff
@@ -0,0 +1,42 @@
+
+export function AboutComponent() {
+    return (
+        <div className={styles.about}>
+            <div className={styles.content}>
+                <img src={picture} alt="About" className={styles.picture} />
+                <div className={styles.content}>
+                    <h1 className={styles.title}>About me</h1>
+                    <div className={styles.description}>
```
It can be more semantic. Pls use section / article instead of div

VasylBiliak committed:
Update About.component.tsx added section instead of div

VasylBiliak committed:
Update About.component.tsx

removed redundant styles.description

MZabolotnev reviewed (changes requested):
Paddings missed in both views ( mobile / desktop )
<img width="613" alt="image" src="https://github.com/VasylBiliak/fe-react-2024/assets/25617217/5beb32f8-7447-4db4-a4e2-50170a113f52">

VasylBiliak committed:
Update about.module.css

added paddings in views mobile and desktop

MZabolotnev reviewed (changes requested):
Still not good. Reason - picture width is hardcoded width: 330px;. You can try width: 100% instead. 
<img width="476" alt="image" src="https://github.com/VasylBiliak/fe-react-2024/assets/25617217/10aeea03-6bf3-4cc6-b728-d77316e89ade">

VasylBiliak committed:
Update about.module.css

Refactored CSS code to enhance responsiveness and readability. Adjusted padding, grid layout, and element styles for better display on various screen sizes. Updated picture border radius and width to ensure consistent design across breakpoints. 

Replaced width: 330px; with width: 100% to ensure adaptability across different screen sizes.
